src/exception/call_stack/elf.cr

```diff
@@ -5,16 +5,19 @@ require "crystal/elf"
 
 struct Exception::CallStack
   protected def self.load_debug_info_impl
+    base_address : LibC::Elf_Addr = 0
```

Comment:
I would avoid using a `TypeDeclaration` here unless necessary:

```suggestion
    base_address = LibC::Elf_Addr.zero
```

Reply:
Later we're passing a pointer to this variable. So IMO it makes sense to pin its type in order to prevent an accidental union type tainting the pointer.